scanner/src/main/kotlin/ScannerFactory.kt

```diff
@@ -28,7 +28,7 @@ import org.ossreviewtoolkit.model.config.ScannerConfiguration
  */
 interface ScannerFactory {
     /**
-     * The name to use to refer to the package manager.
+     * The name to use to refer to the scanner.
```

Comment:
Commit message: For single-file changes the "scope" prefix of the commit message should be the name of that single file (without extension). So here: "ScannerFactory: Fix a wrong code comment".